test/unit/spec/MfaVerify_spec.js

```diff
@@ -2780,6 +2781,103 @@ function (Okta,
         });
       });
     });
+    Expect.describe('Browser back button does not change view', function () {
+      itp('from mfa verify controller', function () {
+        return setupAllFactors().then(function (test) {
+          spyOn(window, 'addEventListener');
+          test.router.start();
+          expectHasRightBeaconImage(test, 'mfa-okta-security-question');
+          test.beacon.dropDownButton().click();
+          clickFactorInDropdown(test, 'SMS');
+          return tick(test);
+        })
+        .then(function (test) {
+          expectHasRightBeaconImage(test, 'mfa-okta-sms');
+          Util.triggerBrowserBackButton();
+          return test;
```

Comment:
I think this should be ```return tick(test);``` so we wait for whatever needs to happen (which in this case is nothing, but that will give enough time in case someone changes the behavior of back in the future.